src/main.rs

```diff
@@ -148,6 +149,35 @@ fn app() -> Result<(), Error> {
         });
     }
 
+    cmds.add("?godbolt flags={} version={} ```\ncode```", |args| {
```

Comment:
I would prefer to not use a closure here unless its necessary.   Please move these into a fn and pass the fn pointer here instead.